### DISCUSSION TITLE
How can I change the attack rate for 1.8?

### DISCUSSION THREAD

**Discussion by CooperSma** [PrismarineJS/mineflayer#2071] [Q&A]
Hey, I am using the mineflayer-pvp plugin and cannot figure out how to change the attack rate, the api docs for this aren't super clear. Is this possible? Thank you in advance!

**Comment by u9g** [comment]
It’s not really made to be changed for spam click pvp, for that you’d have to look at how the source is written with the per weapon cooldown
